lib/core/src/server/cli/utils.ts

```diff
@@ -25,11 +25,11 @@ const warnDLLsDeprecated = deprecate(
 );
 
 export function checkDeprecatedFlags(options: {
-  dlls?: boolean;
+  dll?: boolean;
   uiDll?: boolean;
   docsDll?: boolean;
 }) {
-  if (!options.dlls || options.uiDll || options.docsDll) {
+  if (options.dll || options.uiDll || options.docsDll) {
```

Comment:
shouldn't this be `!options.dll`?

Reply:
I tested it again, after passing `--no-dll`, the `dll` will become `false`, so it should be `option.dll`.

![image](https://user-images.githubusercontent.com/15034155/98108665-5783f600-1ed7-11eb-9650-44d609a2778a.png)

![image](https://user-images.githubusercontent.com/15034155/98108641-4804ad00-1ed7-11eb-8ec8-e639e5a42d05.png)

Reply:
`--no-dll` is no longer needed because the DLLs have been entirely removed, so it should warn the user. if `.dll` is set, it means that the user did not pass `--no-dll` so, so no warning is needed.

Reply:
Got it! I'm already updated.

Reply:
Wait, is it should be `option.noDll`. As u said, the DLLs have been entirely removed. So, in the future, there will not have `dll` option?

Reply:
Thanks! 👍